Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml

```diff
@@ -78,7 +78,7 @@
         </div>
         <div class="govuk-grid-column-full">
             <div class="button-row">
-                <button type="submit" id="continue-button" name="action" value="" class="govuk-button" v-on:click.prevent="checkMergeType">Continue</button>
+                <button type="submit" id="merger-amalgamation-continue-button" name="action" value="" class="govuk-button" v-on:click.prevent="checkMergeType">Continue</button>
```

Comment:
Is it intentional to have both `merger-amalgamation-continue-button` and `amalgamtion-merger-continue-button` (line 161)?